base/math.jl

```diff
@@ -506,7 +506,7 @@ end
 
 Compute the hypotenuse ``\\sqrt{\\sum x_i^2}`` avoiding overflow and underflow.
 """
-hypot(x::Number...) = vecnorm(x)
+hypot(x::Number...) = sqrt(sum(abs2(y) for y in x))
```

Comment:
One of two places which uses a `LinearAlgebra` owned function... I hope this hack is fine for now.